basicsr/models/realesrgan_model.py

```diff
@@ -207,6 +208,8 @@ def optimize_parameters(self, current_iter):
 
         self.optimizer_g.zero_grad()
         self.output = self.net_g(self.lq)
+        if self.cri_artifacts:
```

Comment:
I suggest to use self.cri_ldl, as ldl is the method name of your paper. So using ldl will let readers understand it quickly.